types/mailgun-js/index.d.ts

```diff
@@ -74,6 +75,11 @@ declare namespace Mailgun {
             "recipient-variables"?: BatchSendRecipientVars;
         }
 
+        interface SendTemplateData {
+            "template": string;
```

Comment:
We don't need these quotes here.

```suggestion
            template: string;
```